spec/helpers/find/goto_preview_helper_spec.rb

```diff
@@ -66,6 +66,66 @@ module Find
       end
     end
 
+    describe '#goto_provider_value' do
+      subject do
+        goto_provider_value(param_form_key:, params:)
+      end
+
+      context 'params is empty' do
+        let(:params) { {} }
+
+        it 'returns falsey' do
+          expect(subject).to be_nil
+        end
+      end
+
+      context 'params has goto_provider set to "true"' do
+        let(:params) { { goto_provider: 'true' } }
+
+        it 'returns truthy' do
+          expect(subject).to eq('true')
+        end
+      end
+
+      context 'params has param_form_key with goto_provider set to "true"' do
+        let(:params) { { param_form_key: { goto_provider: 'true' } } }
+
+        it 'returns truthy' do
+          expect(subject).to eq('true')
+        end
+      end
+    end
+
+    describe '#goto_provider?' do
+      subject do
+        goto_provider?(param_form_key:, params:)
+      end
+
+      context 'params is empty' do
+        let(:params) { {} }
+
+        it 'returns falsey' do
+          expect(subject).to be_falsey
+        end
+      end
+
+      context 'params has goto_provider set to "true"' do
+        let(:params) { { goto_provider: 'true' } }
+
+        it 'returns truthy' do
+          expect(subject).to be_truthy
+        end
+      end
+
+      context 'params has param_form_key with goto_provider set to "true"' do
+        let(:params) { { param_form_key: { goto_provider: 'true' } } }
```

Comment:
```suggestion
        let(:params) { { param_form_key: { goto_provider: 'false' } } }
```

Is it worth adding tests for this scenario?